01_fib/anzranjana/main_test.go

```diff
@@ -0,0 +1,96 @@
+package main
+
+import (
+	"bytes"
+	"testing"
+)
+
+func TestMainOutput(t *testing.T) {
+	var buf bytes.Buffer
+	out = &buf
+
+	main()
+
+	expected := "1\n1\n2\n3\n5\n8\n13\n"
+	actual := buf.String()
+
+	if expected != actual {
+		t.Errorf("Unexpected output in main()")
+	}
+}
+
+func TestNegativeFibo(t *testing.T) {
```

Comment:
Did you look at https://goplay.space/#Zy2ta-z8vrO - good example of testing multiple cases rather than having to release the setup.